### PR TITLE
Feature Coding Standards

### PR DESCRIPTION
**Pull request by EtiennePASSOT**
Add coding style define and example 
Add language conventions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,5 +3,140 @@
...
+{
+    return true;
+}
+else
+{
+    return false;
+}
+```
+Here, reading is quite simple in both cases, but good practice in certain languages prefers to use one or the other.
+
```
Comment (Arakhaa) [inline README.md:79]: I find this paragraph good, it addresses in my opinion the expected points.
Perhaps you should conclude by saying that each code standard is to be adopted according to the preference of each team to plan in agreement with everyone upstream.
What do you think ?

File: README.md [hunk 2]
```diff
@@ -3,5 +3,140 @@
...
+* Add a single space after each comma delimiter;
+* Add a single space around binary operators (==, &&, ...), with the exception of the concatenation (.) operator;
+* Use return null; when a function explicitly returns null values and use return; when the function returns void values;
+* Declare public methods first, then protected ones and finally private ones. The exceptions to this rule are the class constructor and the setUp() and tearDown() methods of PHPUnit tests, which must always be the first methods to increase readability;
+
+
+[Read more](https://symfony.com/doc/current/contributing/code/standards.html)
+
+---
+
```
Comment (Arakhaa) [inline README.md:142]: I find the rest good !

**Review by Arakhaa — CHANGES_REQUESTED** [review]
I let you think about the comments I made on your PR _(Feature Coding Standards#1)_ before merge

File: README.md (hunk 1 above)
Comment (EtiennePASSOT) [reply README.md:79]: Good idea, I add a conclusion

**Pull request closed without merging**